src/main/resources/com/google/api/codegen/ruby/smoke_test.snip

```diff
@@ -23,7 +23,15 @@ describe "{@smokeTest.name}" do
       project_id = ENV["SMOKE_TEST_PROJECT"].freeze
     @end
 
-    {@sampleCodeCore(smokeTest.apiMethod)}
+    process_response = ->(response) {
+      @# Nothing to do.
+    }
+
+    {@sampleCodeAliases(smokeTest.apiMethod)}
```

Comment:
Even if you break the sampleCodeCore into 4 different pieces for the reuse purpose of unit test gen, I think we should still provide a simple call entry 'sampleCodeCore', which can be called by smoke test directly, because smoke test does not care about this finer granularity, at least for now.